<launch>
    <arg name="prepare_some_distance" default="0.08" />
    <arg name="record_pose_path" default="$(find rubik_cube_solve)" />
    <arg name="test" default="1" />
    <arg name="speed" default="1" />

    <include file="$(find rubik_cube_solve)/launch/load_pose_path.launch" />

    <param name="/move_group/trajectory_execution/execution_duration_monitoring" type="bool" value="true" />

    <param name="isRuning_solveMagic" type="bool" value="false" />             

    <node pkg="rubik_cube_solve" type="RubikCubeSolve" name="rubik_cube_solve" output="screen" >
        <param name="prepare_some_distance" type="double" value="$(arg prepare_some_distance)" />
        <param name="fakeInitializationState" type="bool" value="true" />
        <param name="record_pose_path" type="str" value="$(arg record_pose_path)" />
        <param name="test" type="int" value="$(arg test)"  />
        <param name="speed" type="double" value="$(arg speed)" />
    </node>

    <node pkg="rubik_cube_solve" type="remove_object" name="rubik_cube_object" output="screen" > 
    </node>
</launch>